gtsfm/multi_view_optimizer.py

```diff
@@ -39,6 +43,13 @@ def __init__(
         self.data_association_module = data_association_module
         self.ba_optimizer = bundle_adjustment_module
         self._run_view_graph_estimator: bool = self.view_graph_estimator is not None
+        self._create_debug_directories()
+
+    def _create_debug_directories(self) -> None:
```

Comment:
Thanks for the PR, Hayk. Question — what’s the definition of a debug output, and how does it differ from other outputs of the system?

Reply:
Hi John, thanks for reviewing. Debug output is the output that we only use for debugging purposes. Other outputs e.g. `results` directory are later used for purposes like 3D visualization.